tests/e2e/features/edit_record.feature

```diff
@@ -55,10 +54,6 @@ Feature: Edit record
             | _locale | nl |
         Then there is element "title" with text "Changed title NL"
 
-        When I visit the "single_record" page with parameters:
```

Comment:
why removed?

Reply:
Because it's incorrect. The correct version is sligthly below, line 67/62.

I assumed this was a copy/paste mistake.